CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ option(
   ENABLE_OPTIMIZER_DEBUG
   "Enable OPTIMIZER_DEBUG when building. Requires Postgres server to be built with OPTIMIZER_DEBUG."
   OFF)
+option(ENABLE_DEBUG_UTILS "Enable debug utilities for the extension." ON)
```

Comment:
Do we need a separate setting? Maybe always enable them in Debug builds?

Reply:
I am also in favor of enabling that in every debug build.

Reply:
They are by default enabled, and since the functions are not defined for non-debug builds, this is how it currently behaves.